MIME-Version: 1.0
Content-Type: application/aegis-patch
Subject: aegis.4.24 - aecp -ind -o multiple user permissions set bug
Content-Name: aegis.4.24.C190.patch
Content-Disposition: attachment; filename=aegis.4.24.C190.patch
X-Aegis-Project-Name: aegis.4.24
X-Aegis-Change-Number: 190

#	
#	A bug has been fixed in the aecp -independent command, it no longer
#	fails with a bug message if you use the --output option.
#	
#	My thanks to Stephen Gross <mrstevegross@gmail.com> for reporting this
#	problem.
#	
#	Signed-off-by: Peter Miller <millerp@canb.auug.org.au>
#	Signed-off-by: Walter Franzini <walter.franzini@gmail.com>
#	
# Aegis-Change-Set-Begin
# QlpoMzFBWSZTWY7TRMgAALhfgAAQUAf//2WiSo6/79/6QAIdtbaYZDSaIyDU0emjVNp6pvVN
# Gh+qeoAA0aBE01PSbTRT0JlPTTUGjTQD1AaeoGgcwAAAAAAAAAAJQgKekBqfqjGk9Ro0AGgN
# ABypvikCqH6iiGwcyQQOz1O3d7eW2JA3DIe6srEiHm+B3uiu70gCHDdvSR2aDJqAkRGjgivM
# w+jLPWzOb+vDuuzsqBfxPNVnVNaYTz4J7bxBzS0DrV94N57RFWu23U+8Py8O61LKXYvJKH5U
# X6SRBa2QFg0mtlClyqKzoAUQYyCLRdFtZgn8MMrNvPfeo9TUghpNd2jpj6SPddqqK2Q0iRBA
# 3+IMXbpXpchDKtmxconot1M8jeIBfFoPgniFFtd49QDZbO+katum6O2HR+C+/KRMPYYVzxWa
# 5rW8QjBGYBXpaFGgRMV3BmnDDogSDbtoQcqJtzmfJ2MWMnV04F+qEYKykiw6mE7XN2KMa2GM
# bDWhVIWtRU/h0ZFdvCdzX37RE4mOfXZqLltUa4V5h5BkYtKKZ10SKUFjaxP8MJX80unZsNWM
# /iDBwi4i+dI9NmK1qi1a7IbzjAzRTfUdcYjE9KT+8snTR7ORxTA77QV0uO8ses2JiklH4HQH
# vK5s9mRjYkkLVWx1S/irQ8LrJo/LRcOZnY0ePyrBgCPCa6rnQuamFRTCIV/ryqcjJX13/2oj
# sGANCiltZgNihSDxYV1TCiachQACXZJ4uXEXw0AEmyC0UOVesgnmWJRM6TVSsccplkoPgGYa
# +WR6a5l0KVKCf4u5IpwoSEdpomQA
# Aegis-Change-Set-End
#
Index: aegis/aecp.cc
--- aegis/aecp.cc
+++ aegis/aecp.cc
@@ -657,8 +657,8 @@
                     mode |= 0111;
                 mode &= ~original_umask;
                 os_chmod(to, mode);
-                os_become_undo();
             }
+            os_become_undo();
         }
         else
         {
